control-plane/connect-inject/endpoints_controller_test.go

```diff
@@ -521,7 +508,6 @@ func TestProcessUpstreams(t *testing.T) {
 				AllowK8sNamespacesSet:  mapset.NewSetWith("*"),
 				DenyK8sNamespacesSet:   mapset.NewSetWith(),
 				EnableConsulNamespaces: tt.consulNamespacesEnabled,
-				EnableConsulPartitions: tt.consulPartitionsEnabled,
```

Comment:
This change looks like it's not intentional. I think we want to keep this and the the corresponding test case param above.